ingress/multi-cluster/mci-https-backend/README.md

```diff
@@ -0,0 +1,385 @@
+# MultiCluster Ingress with https end to end
+
+[Multi-cluster Ingress](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress-for-anthos) for GKE is a cloud-hosted Ingress controller for GKE clusters. It's a Google-hosted service that supports deploying shared load balancing resources across clusters and across regions.
+
+[Multi-cluster service](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-ingress#multiclusterservice_resources) is a Google developed [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (Custom Resource Definition) for GKE. MCS is a custom resource used by Multi Cluster Ingress that is a logical representation of a Service across multiple clusters. It also allows to use [HTTPS backends](hhttps://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-ingress#application_protocols) with MCI.
+
+## Use-cases
+
+- Encrypted traffic between both clients and GKE Ingress and GKE Ingress and Backend services
```

Comment:
GKE Ingress is technically not used here. Can you rephrase this to "Encrypted traffic between clients and HTTPS LoadBalancer and  from the LoadBalancer to Backend services" please ?

Reply:
Really a silly mistake. Corrected it now.